x-pack/plugins/transform/public/app/sections/create_transform/components/group_by_list/popover_form.tsx

```diff
@@ -298,16 +298,17 @@ export const PopoverForm: React.FC<Props> = ({ defaultData, otherAggNames, onCha
       {isUnsupportedAgg && (
         <>
           <EuiSpacer size="m" />
-          <EuiCodeEditor
-            mode="json"
-            theme="textmate"
-            width="100%"
-            height="200px"
-            value={JSON.stringify(getEsAggFromGroupByConfig(defaultData), null, 2)}
-            setOptions={{ fontSize: '12px', showLineNumbers: false }}
-            isReadOnly
-            aria-label="Read only code editor"
-          />
+          <EuiCodeBlock
+            aria-label={i18n.translate('xpack.transform.agg.popoverForm.codeBlock', {
+              defaultMessage: 'Read only code editor',
```

Comment:
Likewise, not sure if this is the right aria label for this component

Reply:
Updated in 5bb656e8433a3c0c2ef4cefb3f5f5835c9158cc1